### PR TITLE
Timeout bug fix

### PR DESCRIPTION
**Pull request by BrendanBeachBC**
# Description

This PR includes the following proposed change(s):

FLA-1456
- Fixes bug where axios call for submission waited full 60 seconds every time
- Adds feature flag for rush processing tab

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit tests, manual testing.

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/.env.template [hunk 1]
```diff
@@ -6,3 +6,4 @@ REACT_APP_API_BASE_URL=http://localhost:8080
 REACT_APP_BAMBORA_REDIRECT_URL=http://localhost:3000/efilinghub
 REACT_APP_BCSC_USERINFO_URL=http://localhost
 REACT_APP_CSO_BASE_URL=http://localhost/cso
+REACT_APP_RUSH_TAB_FEATURE_FLAG=false
```
Comment (TayGov) [inline src/frontend/efiling-frontend/.env.template:9]:
```suggestion
REACT_APP_RUSH_TAB_FEATURE_FLAG=false

```

Comment (TayGov) [inline src/frontend/efiling-frontend/.env.template]:
```suggestion
REACT_APP_RUSH_TAB_FEATURE_FLAG=false

```

**Review by TayGov — APPROVED** [review]
👍

File: src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js [hunk 2]
```diff
@@ -64,13 +64,18 @@ const determineIfProtectionOrder = (documents) => {
 };
 
 export default function PackageReview() {
+  const rushTabFeatureFlag = window.env
+    ? window.env.REACT_APP_RUSH_TAB_FEATURE_FLAG
+    : process.env.REACT_APP_RUSH_TAB_FEATURE_FLAG;
```
Comment (KDNeufeld) [inline src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js:69]: Would be nice if this went in EnvConfig.js to consolidate all env vars.

Comment (BrendanBeachBC) [reply src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js:69]: Good point, I didn't look at the EnvConfig.js file when I created this. This is an old PR but definitely something we should update in the future.

**Pull request merged**